process_report/invoices/nonbillable_invoice.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+
+import process_report.invoices.invoice as invoice
+
+
+@dataclass
+class NonbillableInvoice(invoice.Invoice):
+    nonbillable_pis: list[str]
+    nonbillable_projects: list[str]
+
+    def _prepare_export(self):
+        self.data = self.data[
+            self.data[invoice.PI_FIELD].isin(self.nonbillable_pis)
+            | self.data[invoice.PROJECT_FIELD].isin(self.nonbillable_projects)
+        ]
```

Comment:
how come we don't implement a ` _process` for this class?

Reply:
The only thing needed to make the nonbillable invoice from the merged csv was to filter out nonbillable PIs and projects. I thought this would fit into the `_prepare_export` function instead of `_process`, since there wasn't any real computations being performed, only basic filtering. 

I guess this boiled down to a small design choice for me.

Reply:
@QuanMPhm you are absolutely right, we only need to merge so there's nothing to process.